app/models/solidus_subscriptions/failure_dispatcher.rb

```diff
@@ -3,7 +3,7 @@
 module SolidusSubscriptions
   class FailureDispatcher < Dispatcher
     def dispatch
-      order.touch :completed_at
+      # order.touch :completed_at
```

Comment:
Could we add a comment referencing this PR?

Reply:
I replaced them with calls to touch `updated_at`. It's probably moot but it's a good way to be sure.